src/Metadata/Extractor/XmlExtractor.php

```diff
@@ -43,15 +43,43 @@ protected function extractPath(string $path)
                 'shortName' => $this->phpize($resource, 'shortName', 'string'),
                 'description' => $this->phpize($resource, 'description', 'string'),
                 'iri' => $this->phpize($resource, 'iri', 'string'),
-                'itemOperations' => $this->getAttributes($resource, 'itemOperation') ?: null,
-                'collectionOperations' => $this->getAttributes($resource, 'collectionOperation') ?: null,
+                'itemOperations' => $this->getOperations($resource, 'itemOperation'),
+                'collectionOperations' => $this->getOperations($resource, 'collectionOperation'),
                 'attributes' => $this->getAttributes($resource, 'attribute') ?: null,
                 'properties' => $this->getProperties($resource) ?: null,
             ];
         }
     }
 
     /**
+     * Return the array containing configured operations. Returns NULL if there is no operation configuration.
+     *
+     * @param \SimpleXMLElement $resource
+     * @param string            $operationType
+     *
+     * @return array|null
+     */
+    private function getOperations(\SimpleXMLElement $resource, string $operationType)
+    {
+        if ($legacyOperations = $this->getAttributes($resource, $operationType)) {
+            @trigger_error(
+                sprintf('Configuring "%1$s" tags without using a parent "%1$ss" tag is deprecrated since API Platform 2.3 and will not be possible anymore in API Platform 3', $operationType),
```

Comment:
@dunglas What version should we use for deprecation? I've the same in #904 but I'm not sure what version we need in the "`since API Platform 2.3`".